test/unit/test_version.py

```diff
@@ -66,8 +105,32 @@ def test_installed_version_lower(self, mock_get):
     @patch('sys.stderr')
     @patch('dbt.version.requests.get')
     def test_dbt_version_flag(self, mock_get, stderr):
-        mock_get.return_value.json.return_value = {'info': {'version': '0.10.1'}}
+        mock_get.return_value.json.return_value = {
+            'info': {'version': '0.10.1'}
+        }
 
         with self.assertRaises(SystemExit) as exc:
             dbt.main.handle_and_check(['--version'])
         self.assertEqual(exc.exception.code, 0)
+
+    @patch('importlib.util.find_spec', autospec=True)
+    @patch('importlib.import_module', autospec=True)
+    def test_get_dbt_plugins_info_with_version_info(
```

Comment:
What do you suggest to integration test it ?

Reply:
We don't currently integration test, so I think we can just live without it.